docs/design.md

```diff
@@ -0,0 +1,59 @@
+# Spark Operation Design
```

Comment:
Spark Operation or Spark Operator Design?

Reply:
Hmm, typos. Should be `Operator`.